Streetcode/Streetcode.WebApi/Program.cs

```diff
@@ -13,6 +16,14 @@
 builder.Services.ConfigurePayment(builder);
 builder.Services.ConfigureInstagram(builder);
 
+builder.Services.AddMemoryCache();
+builder.Services.Configure<IpRateLimitOptions>(builder.Configuration.GetSection("IpRateLimiting"));
+builder.Services.AddSingleton<IIpPolicyStore, MemoryCacheIpPolicyStore>();
+builder.Services.AddSingleton<IRateLimitCounterStore, MemoryCacheRateLimitCounterStore>();
+builder.Services.AddSingleton<IRateLimitConfiguration, RateLimitConfiguration>();
+builder.Services.AddSingleton<IProcessingStrategy, AsyncKeyLockProcessingStrategy>();
+builder.Services.AddInMemoryRateLimiting();
+
```

Comment:
it is better to move code in another method and use it here, as we setup other services

Reply:
Done